src/main/java/org/spongepowered/common/config/category/LoggingCategory.java

```diff
@@ -179,6 +181,10 @@ public void setLogEntitySpeedRemoval(boolean flag) {
         this.logEntitySpeedRemoval = flag;
     }
 
+    public boolean logTransactionMergeFailure() {
```

Comment:
I know @gabizou said "could this be configurable?" and attached it to the log line, but I'm not sure this is what he meant - I think we should _always_ be reporting a failure to the log.

Was the meaning that this entire behaviour on failure should be configurable - as in, original vs replacement?

Reply:
Well this and whether silencing after X number of failed/merged transactions happens.

Reply:
I can understand the configuration to "silence after X messages", but why would the behavior of the failure need to be configurable? I can't imagine a server owner changing that.